Doppler.AccountPlans/Controllers/AccountPlansController.cs

```diff
@@ -45,57 +49,60 @@ public class AccountPlansController
             [FromQuery] int discountId,
             [FromQuery] string promocode = null)
         {
-            _logger.LogInformation("Calculating plan amount details.");
-
-            var newPlan = await _accountPlansRepository.GetPlanInformation(newPlanId);
-            var currentPlan = await _accountPlansRepository.GetCurrentPlanInformation(accountName);
-            var discountPlan = await _accountPlansRepository.GetDiscountInformation(discountId);
+            using (_timeCollector.StartScope())
+            {
```

Comment:
I think that could be possible avoiding to use `{` to reduce the format changes

Reply:
sure, i'll make the change so everything has the same format